src/structlog/stdlib.py

```diff
@@ -330,6 +330,14 @@ def filter_by_level(logger, name, event_dict):
         raise DropEvent
 
 
+def add_log_level_number(logger, method_name, event_dict):
+    """
+    Adds the log level number to the event dict.
```

Comment:
Kind of as above: it should broadly explain what that is, maybe even link to the docs.

Also: `Add` not `Adds` please.